share/scripts/inverse/run_gromacs.sh

```diff
@@ -115,12 +119,42 @@ if [[ ${CSG_MDRUN_OPTS} ]]; then
   mdrun_opts+=" ${CSG_MDRUN_OPTS}"
 fi
 
-#see can run grompp again as checksum of tpr does not appear in the checkpoint
-critical ${grompp[@]} -n "${index}" -f "${mdp}" -p "$topol_in" -o "$tpr" -c "${conf}" ${grompp_opts} 2>&1 | gromacs_log "${grompp[@]} -n "${index}" -f "${mdp}" -p "$topol_in" -o "$tpr" -c "${conf}" ${grompp_opts}"
-[[ -f $tpr ]] || die "${0##*/}: gromacs tpr file '$tpr' not found after runing grompp"
+# run gromacs multiple or one time
+if [[ ${multi} ]]; then
+  for((i=0;i<${multi};i++)); do
+    for j in index mdp topol_in conf; do
+      # read $j.$i (e.g index.1} and use ${!j} (e.g. ${index} ) as default and store it in f
```

Comment:
Is it really $i.$j ?  Or should it be $i.$j.dat; I was thinking on sth. like index.1.ndx ?
I anyway have problems in running this script. It says It cannot find the mdp.files, although I've specified them under filelist. So is this a bug or a problem related to the .xml file ?

Reply:
I think the comment is misleading, it should be `cg.inverse.gromacs.${j}.sim${i}`, where `$j` is the word `index`, `mdp` etc and `$i` is 0..n, so e.g. `cg.inverse.gromacs.mdp.sim2` is read.

Reply:
And @DRosen285, do that work https://github.com/votca/csg-tutorials/blob/dc911a7a828a8541d1c96d4d38eb1590f0609c58/methanol/imc_gromacs-remd/settings.xml#L63

Reply:
Great. Now it's clear and everything is working :)

Reply:
Hi, in the following piece I do not understand - why the file with suffix '1' is missing, i.e. no 'sim1'?
`      <mdp>
        <sim0>grompp0.mdp</sim0>
        <sim2>grompp2.mdp</sim2>
        <sim3>grompp3.mdp</sim3>
      </mdp>
`

Reply:
You could add `<sim1>grompp.mdp</sim1>`, but `grompp.mdp` is the default anyhow.

Reply:
Plus, you would need to set the normal `<mdp>` option to tell VOTCA where look up default settings in `get_simulation_setting()`.

So:
```
    <mdp>grompp1.mdp
        <sim0>grompp0.mdp</sim0>
        <sim1>grompp1.mdp</sim1>
        <sim2>grompp2.mdp</sim2>
        <sim3>grompp3.mdp</sim3>
      </mdp>
```